### ISSUE TITLE
Accessibility issues flagged from the Congress on Accessibility inclusion.

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3745]
Reported by Sean Morton (sean.morton@hrsdc-rhdcc.gc.ca) who created a feedback form used by participants at the Canadian Congress on Accessibility Inclusion.

- “This form doesn't work in large print. The captcha button is covering most of the text when I zoom in.”
- "The French text in this message is not properly tagged so my screen reader (JAWS) does not switch automatically."

**Comment by jose-cds** [comment]
Form in question

https://forms-formulaires.alpha.canada.ca/en/id/clwf6o5ho012cx883w47n21dd

**Comment by srtalbot** [comment]
Update: There is no more recaptcha symbol, but good to consider for hCATPCHA implementation @thiessenp-cds 

Just need to check into the French text tags on JAWS.

**Comment by timarney** [comment]
Is there a dev action item on this ?

**Comment by Abi-Nada** [comment]
To validate that it works correctly now (as someone who doesn't have JAWS), I checked to see if there's a `lang="fr"` attribute in the HTML + confirmed with Pete

Closing this issue as we've confirmed there's French text tags
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL2ZMQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4d86ad3a50b1bf976d228cfd69496ac7a5b7fac1/Screenshot%202025-02-07%20at%2010.48.16%E2%80%AFAM.png" alt="Screenshot 2025-02-07 at 10.48.16 AM.png" />
